src/main/java/io/cdap/plugin/dlp/SensitiveRecordRedaction.java

```diff
@@ -87,6 +90,10 @@
   // DLP service client for managing interactions with DLP service.
   private DlpServiceClient client;
 
+  // Required fields that need to be sent to DLP for the transform to work, this variable is used to cache the set
+  // since it will not change during the execution of the plugin
+  private HashSet<String> requiredFields = null;
```

Comment:
Have the type as `Set`. Also, field are by default `null`, so no need to initialize as `null`. BTW, it's better not to use `null` but an empty Set to denote nothing.

Reply:
Done